Source/DafnyLanguageServer/Workspace/IdeState.cs

```diff
@@ -25,6 +25,7 @@ public record IdeVerificationResult(VerificationPreparationState PreparationProg
 /// </summary>
 public record IdeState(
   int Version,
+  ISet<Uri> OwnedUris, // TODO include project Uri
```

Comment:
What is this TODO exactly?

Reply:
It's resolved now, by this line: https://github.com/dafny-lang/dafny/pull/4770/files#diff-1c97a34c28a0bbbc6560c4c5b1649d777a9b200abf19099af19f73601e48228cR32